src/middlewared/middlewared/plugins/disk_/swap_mirror_freebsd.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+from bsd import geom
+from copy import deepcopy
+
+from middlewared.service import CallError, Service
+from middlewared.utils import filter_list, run
+
+from .swap_mirror_base import DiskSwapMirrorBase
+
+
+class DiskService(Service, DiskSwapMirrorBase):
+
+    async def create_swap_mirror(self, name, options):
+        cp = await run('gmirror', 'create', name, *(options['paths']), check=False, encoding='utf8')
```

Comment:
Not important, but why *(options['paths']) and not *options['paths']

Reply:
No reason at all, thank you for pointing it out